client/src/pages/student-info/components/StudentDetails.tsx

```diff
@@ -25,7 +29,7 @@ function StudentDetails({ student, ...props }: Props): JSX.Element {
           </TableRow>
           <TableRow hover>
             <TableCell>Matrikelnummer</TableCell>
-            <TableCell>{student.matriculationNo ?? 'Keine Matrikelnummer'}</TableCell>
+            <TableCell>{student.matriculationNo || 'Keine Matrikelnummer'}</TableCell>
```

Comment:
Why do you use `||` instead of `??`? To catch empty strings, aswell?

Reply:
yes  
if it was only left out for some students, I got an empty string instead of undefined (which I guess makes sense).
In this case, I still wanted to display the message.